src/style.rs

```diff
@@ -366,6 +366,16 @@ impl Command for SetStyle {
 
         Ok(())
     }
+
+    #[cfg(windows)]
```

Comment:
If you rebase, i'll approve and merge. Thanks!

Reply:
Done!